src/Bicep.Core/Semantics/Namespaces/SystemNamespaceSymbol.cs

```diff
@@ -389,6 +391,67 @@ public class SystemNamespaceSymbol : NamespaceSymbol
                 .WithReturnType(LanguageConstants.String)
                 .WithDescription("Returns a value in the format of a globally unique identifier. **This function can only be used in the default value for a parameter**.")
                 .WithFlags(FunctionFlags.ParamDefaultsOnly)
+                .Build(),
+
+            new FunctionOverloadBuilder("loadContent")
+                .WithReturnType(LanguageConstants.String)
```

Comment:
>String [](start = 50, length = 6)

So we will always return the `string` type? Can we make it return the string literal type of the value instead?

Reply:
String literal has a flaw that it's name is the content, and it might be looking very odd for a large files in VSCode when you hover after you assign to a variable, but perhaps that is fine.

Reply:
Handling it as a string literal and displaying the type with truncation logic would be my preferred approach:
1. It's going to provide a visual indicator on hover to the user for how their file is being interpreted by Bicep. For example - if you've picked the wrong encoding, or have strange whitespacing, it'll at least give you something to look at to get a feel for whether you've done something incorrect.
1. It'll allow us to do type transformations in the future - e.g. to read arbitrary JSON files and provide type validation:
    ```bicep
    var myFile = json(loadContent('./myFile.json')) // if we imagine 'myFile.json' contains '{"validProp": "stringVal" }'
    var working = myFile.validProp // Permitted, and we can infer the type of "validProp" correctly
    var nonWorking = myFile.missingProp // Error here
    ```

Reply:
@anthony-c-martin this is how it will look like in vscode when using string literal:
![image](https://user-images.githubusercontent.com/7921224/114070826-b68fd180-98a0-11eb-99c3-4ba98ca02a27.png)